.github/workflows/install-mpw-6c.yaml

```diff
@@ -0,0 +1,47 @@
+name: build Sky130 GDS
+# either manually started, or on a schedule
+on: [ push ]
+jobs:
+  install:
+    env:
+        OPENLANE_ROOT:  /home/runner/openlane
+        OPENLANE_IMAGE_NAME:    efabless/openlane:2022.02.23_02.50.41
+        PDK_ROOT:       /home/runner/pdk
+        PDK:            sky130A
+
+    # ubuntu
+    runs-on: ubuntu-latest
+    steps:
+
+    # checkout the repo to get the scripts
+    - uses: actions/checkout@v2
+
+    # build the user project and wrapper
+    - name: pdk & openlane
+      run: |
+        cd $HOME
+        git clone -b mpw-6c https://github.com/efabless/caravel_user_project.git
+        cd caravel_user_project/
+        make setup
+
+    - name: make GDS
```

Comment:
Curious what's missing from https://github.com/The-OpenROAD-Project/actions#openlane_run to be useful here.

(I suspect the inability to specify a different branch for the design and for openlane might be one: @vvbandeira)

Reply:
thanks for the review Proppy! Also I need to check this action, I haven't read it yet.